src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getComments(postId: number) {
```

Comment:
Please rename it to getPostComments and add the type for response: `Promise<Comment[]>`.
```suggestion
export async function getPostComments(postId: number): Promise<Comment[]> {
```